### ISSUE TITLE
Πρόβλημα phpCAS

### ISSUE THREAD

**Issue by vbanos** [AlexandriaUP/Alexandria#2]
Αφού έγινε η εγκατάσταση, στην αρχική σελίδα, όταν πάμε "Σύνδεση ως μέλος ΔΕΠ" καλείται το εξής URL:
https://alexandria.ihu.edu.gr/uppa_core/saml/paperaggregator/index.php?sso
και παίρνουμε το εξής error:
```
phpCAS error: phpCAS::setCasServerCACert(): CAS_InvalidArgumentException: Certificate file does not exist CAS_Request_CurlRequest in /home/alexandria/public_html/uppa_core/saml/paperaggregator/index.php on line 146
```

Τί να κάνουμε για να μην γίνεται αυτό; Βλέπω παρακάτω το σχόλιο που λέει ότι θα μπορούσαμε να το απενεργοποήσουμε.
```
    phpCAS::setCasServerCACert($cas_server_ca_cert_path);
    //phpCAS::setCasServerCACert($cas_server_ca_cert_path, false);
    // For quick testing you can disable SSL validation of the CAS server.
    // THIS SETTING IS NOT RECOMMENDED FOR PRODUCTION.
    // VALIDATING THE CAS SERVER IS CRUCIAL TO THE SECURITY OF THE CAS PROTOCOL!
    //phpCAS::setNoCasServerValidation();
```
Εικάζω ότι λείπει το configuration `cas_server_ca_cert_path` από το `settings.php`.

Ευχαριστώ.

**Comment by AlexandriaUP** [comment]
Ναι, σε δοκιμαστικό περιβάλλον μπορεί να απενεργοποιηθεί. Σε παραγωγικό περιβάλλον θα πρέπει για λόγους ασφαλείας να είναι ενεργοποιημένο. Στο settings.php θα πρέπει να ορίσετε την τοποθεσία όπου θα έχετε αποθηκεύσει το CA chain για το πιστοποιητικό του CAS Server. Αν επισκεφθείτε τον CAS server από τον browser και πατήσετε πάνω στο εικονίδιο της κλειδαριάς και επιλέξετε να δείτε το πιστοποιητικό, ο firefox έχει μια επιλογή "PEM (αλυσίδα)" που επιτρέπει να κατεβάσετε το αρχείο που χρειάζεστε.
